### ISSUE TITLE
[UI/UX] imrpovements suggestions

### ISSUE THREAD

**Issue by gzuuus** [nostrband/nostr-universe#84]
The interface is very good and shows you the information properly, however after having been using the application for a while, I have thought that there are some details that could make the user experience better. 

My first suggestion is that the trending notes that appear in the first row could display the full truncated content without having to open a client again to see the content, many times it is a bit frustrating to have to navigate to another application just to see the content of the note, and then go back to continue reading the other trending notes, due to the loading time and the redundancy of the information. If the content can be displayed with a 'show more' button for example, it would be more pleasant to navigate and finally if you want to interact open it in a client.

My second suggestion is related to the first one, and that is that although there is already the option to open the note in a native client, it would be great if this option would appear without having to load the whole list of nip89 compatible applications. It would be great if this option could appear without the need to load anything, because it is not really necessary to load anything to show this option since the nip19 identifier is already there.

**Comment by nostrband** [comment]
Thanks for great observations and suggestions.

Your second suggestion is implemented in the new v0.2.0, please check it out.

**Comment by gzuuus** [comment]
Yes, it works as expected, now I can navigate very fast between Spring and my native Nostr app 👏👏.
On the other hand, I have noticed a decrease in the performance of the app, it runs slower.

**Comment by gzuuus** [comment]
In the latest version (0.4), the ability to instantly show the open button in the native application has been lost.
Also, I still think that the functionality to show all the content of the notes that appear in the 'new' section could be a good addition to improve the user experience.

**Comment by nostrband** [comment]
> In the latest version (0.4), the ability to instantly show the open button in the native application has been lost. 

If you select Native app once, it should then be auto-selected and "Open with" should change to "Open with Native app" - and it's as accessible as it was before. Is that flow not working for you?

> Also, I still think that the functionality to show all the content of the notes that appear in the 'new' section could be a good addition to improve the user experience.

I agree that a preview would be nice, looking into various ways to provide that.

**Comment by gzuuus** [comment]
Yes, opening with native app appears, but you have to wait for the nip89 app list to load, which is sometimes slow and unnecessary if you want to open in native app. Now that there is a menu before clicking on a note where you can zap, see answers etc, it might be interesting to simply add a button to open in native app, plus another button to open with another app.

For the preview I think the easiest way is to add a show more/less button and then show the preview in full screen or simply expand the content using the same container.

**Comment by nostrband** [comment]
@gzuuus could you please try v0.8.0 it adds event previews

**Comment by gzuuus** [comment]
I tried the other day! i was enjoying it! thanks so much!!
